src/pybel/schema/node_types/variants/base_variant.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema#",
+    "title": "BEL Variant Node",
+    "description": "Basic schema to validate JSON files describing a BEL variant object",
+    "type": "object",
+    "properties": {
+      "concept": {
+        "description": "Entity description if the variant is a protein or gene modification",
+        "$ref": "../../entity.schema.json"
+      },
+      "code": {
+        "description": "Amino acid code for the affected residue in a protein modification",
+        "enum": [
+          "Ala", "Arg", "Asn", "Asp", "Cys", "Glu", "Gln",
+          "Gly", "His", "Ile", "Leu", "Lys", "Met", "Phe",
+          "Pro", "Ser", "Thr", "Trp", "Tyr", "Val"
+        ]
+      },
+      "pos": {
+        "description": "Position of the affected residue in a protein modification",
+        "type": "number"
+      },
+      "hgvs": {
+        "description": "HGVS variant string",
```

Comment:
oooooo HGVS is its own language. I think you can have a pass on trying to validate it :)

Reply:
@aman527 double check this, HGVS is much more expansive than just substitutions. You should not try writing out a pattern for this, but just accept whatever comes

Reply:
Just removed the regex pattern